src/process/helpers.h

```diff
@@ -117,3 +118,26 @@ static __always_inline int fill_syscall(syscall_info_t *syscall_info, void *ts,
 
     return 0;
 }
+
+// it's argument should be a pointer to a file
```

Comment:
Silly pedantic comment:  The apostrophe technically shouldn't be here; the rules for both marking contractions and marking possession call for it, but the disambiguation rule for this situation says that the contraction wins the apostrophe.